package.json

```diff
@@ -37,12 +37,13 @@
     "docs": "repo-tools exec -- jsdoc -c .jsdoc.js",
     "lint": "repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
     "prettier": "repo-tools exec -- prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
```

Comment:
do we need `gts check` somewhere in the scripts?

Reply:
Good question :) My plan was to land this first.  `gts fix` is going to make the changeset *huge*, and I want to isolate that to one PR.